### PR TITLE
doc: nrf: lib: bootloader

### PR DESCRIPTION
**Pull request by richabp**
Created a separate Bootloader libraries section under Libraries page and moved Bootloader pages out of Other libraries.

Signed-off-by: Richa Pandey <richa.pandey@nordicsemi.no>

### CODE REVIEW

File: doc/nrf/libraries/bootloader/index.rst [hunk 1]
```diff
@@ -0,0 +1,11 @@
+.. _lib_bootloader:
+
+Bootloader libraries
+#####################################
```
Comment (greg-fer) [inline doc/nrf/libraries/bootloader/index.rst:4]: The header formatting line should match the length of the header title.

**Comment by greg-fer** [comment]
I think it makes sense to add an entry to the changelog (doc/nrf/releases/release-notes-changelog.rst) in its documentation section to highlight this change and help users who would look for these libraries under Other libraries. Poke me if you need help with updating this file.

File: doc/nrf/libraries/bootloader/index.rst (hunk 1 above)
Comment (richabp) [reply doc/nrf/libraries/bootloader/index.rst:4]: yes, Thanks for pointing it out. Bartek just mentioned this.

**Review by hakonfam — APPROVED** [review]
nit: all commits to this repository should have a JIRA ticket ref typically in the format

Ref: NCSDK-99999

**Review by greg-fer — APPROVED** [review]

**Review by b-gent — APPROVED** [review]

**Pull request merged**